city_scrapers/spiders/chi_police_retirement.py

```diff
@@ -0,0 +1,129 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiPoliceRetirementSpider(CityScrapersSpider):
+    name = "chi_police_retirement"
+    agency = "Policemen's Annuity and Benefit Fund of Chicago"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.chipabf.org"]
+    start_urls = ["http://www.chipabf.org/ChicagoPolicePension/MonthlyMeetings.html"]
+    TAG_RE = re.compile(r'<[^>]+>')
+
+    def parse(self, response):
+        year = self._parse_year(response)
+        date_table = response.xpath('//*[@id="content0"]/div[3]/table')
+        date_items = date_table.extract()[0].split('<tr>')
+
+        skip_first_item = True
+        for item in date_items:
+            if skip_first_item:
+                skip_first_item = False
+                continue
+
+            meeting = Meeting(
+                title=self._parse_title(response),
+                description=self._parse_description(response),
+                classification=self._parse_classification(response),
+                start=self._parse_start(item, year),
+                time_notes=self._parse_time_notes(),
+                end=self._parse_end(),
+                all_day=False,
+                location=self._parse_location(response),
+                source=self._parse_source(response),
+                links=self._parse_links(item)
+            )
+
+            # replace before this
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "Board Meetings of The Policemen\'s Annuity & Benefit Fund"
+
+    def _parse_description(self, response):
+        """Parse or generate meeting description."""
+        raw_description = ''.join(response.xpath('//*[@id="content0"]/p/text()').extract())
+        clean_description = self._clean_escape_chars(raw_description)
+        print(repr(' '.join(clean_description.split())))
+        return ' '.join(clean_description.split())
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return BOARD
+
+    def _parse_start(self, item, year):
+        start = self._get_date_string(item, year)
+        return datetime.strptime(start, '%B %d %Y %I%p')
+
+    def _parse_time_notes(self):
```

Comment:
You can remove this and `_parse_end` if there isn't any functionality in them to keep things more simple. Not essential though, just ends up being easier to read